packages/docusaurus-theme-classic/src/index.js

```diff
@@ -55,7 +55,11 @@ module.exports = function(context, options) {
     },
 
     getClientModules() {
-      return ['infima/dist/css/default/default.css', customCss];
+      return [
+        'infima/dist/css/default/default.css',
+        'remark-admonitions/styles/infima.css',
```

Comment:
It bothers me a bit to use Infima variables directly... what if we create new custom properties for each of them? I mean the following:

Before:

```css
.admonition {
  background-color: var(--ifm-alert-background-color);
 // ...
}
```

After:

```css
:root {
  --ra-alert-background-color: var(--ifm-alert-background-color);
}

.admonition {
  background-color: var(--ra-alert-background-color);
  // ...
}
```

We still rely on Infima, but in the event of breaking changes, we can easily fix something. Or can we better define our own styles? I want to know the opinion from @yangshun about this.

Reply:
https://github.com/elviswolcott/remark-admonitions/pull/16
I've put up a PR to make this change and clean up the styles. I can merge and publish a new version if this seems good.

Reply:
LGTM but let's wait for the reaction from @yangshun

Reply:
I agree with @lex111's suggestion.

Reply:
Sounds good. I'll merge and release those changes.

Reply:
I've released [v1.1.1](https://github.com/elviswolcott/remark-admonitions/releases/tag/v1.1.1) with these changes